specifications/M1/credential.issuance.protocol.md

```diff
@@ -0,0 +1,313 @@
+
+```json
+{
+  "format": "ldp_vc",
+  "@context": {
+    "ssi": "https://w3id.org/edc/v0.8/",
+    "odrl": "https://www.w3.org/ns/odrl.jsonld/",
+    "bindingMethods": "ssi:bindingMethods",
+    "cryptographicSuites": "ssi:cryptographicSuites",
+    "issuancePolicy": "ssi:issuancePolicy",
+    "offerReason": "ssi:offerReason",
+    "credentialSubject": "ssi:credentialSubject",
+    "types": "ssi:types"
+  },
+  "types": [
+    "VerifiableCredential",
+    "CompanyCredential"
+  ],
+  "offerReason": "reissue",
+  "bindingMethods": [
+    "did:web"
+  ],
+  "cryptographicSuites": [
+    "JsonWebKey2020"
+  ],
+  "issuancePolicy": {
+    "permission": [
+      {
+        "action": "use",
+        "constraint": {
+          "and": [
+            {
+              "leftOperand": "ssi:CredentialPrereq",
+              "operator": "eq",
+              "rightOperand": "active"
+            }
+          ]
+        }
+      }
+    ]
+  }
+}
+```
+
+> TODO: Define SSI context and include term alias that are shown above. Also, need to define a way to
+> associate `issuancePolicy` to an ODRL policy so that Json-Ld expansion is performed properly on sub-nodes.
+
+# 5. Issuer Metadata endpoint
+
+A credential issuer MUST support the Issuer Metadata endpoint using the HTTPS scheme and the `GET method`. The URL of
+the endpoint is the base issuer url with the appended path `/.well-known/vci`.
+
+The response is a JSON object with the following properties:
+
+- `credentialIssuer`: REQUIRED. A unique identifier of the issuer, for example, a DID.
+- `credentialsSupported`: OPTIONAL. A Json Array containing a list of Json Objects with properties corresponding
+  to [Credential Offer Parameters](#credential-offer-parameters).
+
+```json
+{
+  "credentialIssuer": "did:web:issuer-url",
+  "credentialsSupported": [
+    {
+      "types": [
+        "VerifiableCredential",
+        "CompanyCredential"
+      ],
+      "offerReason": "reissue",
+      "bindingMethods": [
+        "did:web"
+      ],
+      "cryptographicSuites": [
+        "JsonWebKey2020"
+      ],
+      "issuancePolicy": {
+        "permission": [
+          {
+            "action": "use",
+            "constraint": {
+              "and": [
+                {
+                  "leftOperand": "ssi:CredentialPrereq",
+                  "operator": "eq",
+                  "rightOperand": "active"
+                }
+              ]
+            }
+          }
+        ]
+      }
+    }
+  ]
+}
+
+```
+
+# 6. Credential Request Status Endpoint
+
+The issuer MUST provide an `HTTPS GET` endpoint for retrieving the status of a credential at the base issuer url with
+the appended path `/requests/<request id>`. The issuer SHOULD implement access control such that only the client that
+made the request may access a particualr request status.
+
+If accepted, a Json object with a `status` property set to one of the following
+values: `RECEIVED` | `REJECTED` | `ISSUED`.
```

Comment:
verb missing.

Reply:
Not sure I follow. Status should be past tense states.

Reply:
```suggestion
If accepted, a Json object with a `status` property set to one of the following
values: `RECEIVED` | `REJECTED` | `ISSUED` will be returned.
```